extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -181,6 +187,9 @@ public String updateApiUrl(String original) {
     static final class SwaggerUiConfig {
         /**
          * The path of the swagger-ui servlet.
+         * <p>
+         * The value `/` is not allowed as it blocks the application from serving anything else.
+         * </p>
```

Comment:
Ah ah, that's why I was precise about just adding a `<p>` before :). You don't need an ending p and Javadoc throws a warning for that.

Reply:
:-) Let me change that.